docs/installation-deb.md

```diff
@@ -25,7 +23,7 @@ instance or a cluster.  Metrictank will also respond to queries: if the data is
 RAM, and older data is fetched from cassandra.  This happens transparantly.
 Metrictank maintains an index of metrics metadata, for all series it sees.  
 You can use an index entirely in memory, or backed by Cassandra for persistence.
-You'll typically query metrictank by querying graphite-api which uses the graphite-metrictank plugin to talk
+You'll typically query metrictank by querying graphite which uses the clustering capabilities of graphite to talk
 to metrictank.  You can also query metrictank directly but this is experimental and too early for anything useful.
```

Comment:
"You can also query metrictank directly but this is experimental and too early for anything useful." this is not true anymore. in fact querying MT directly is now the preferred way. Let's replace this sentence, along with the "You'll typically..." sentence by something like: "You can query metrictank directly (it has fast, but limited built-in processing and will fallback to graphite when needed) or you can also just query graphite which will always use graphite's processing but use metrictank as a datastore" except I'm not sure if the latter statement is true.